.github/workflows/ci-rerun.yml

```
@@ -0,0 +1,105 @@
+name: Re-run Workflow
+
+on:
+  workflow_run:
+    workflows:
+      - CI
+    types:
+      - completed
+  pull_request_target:
+    types:
+      - closed
+      - labeled
+      - unlabeled
+  schedule:
+    - cron: '30 */3 * * *' # every 3 hours (30 minutes past the hour)
+  workflow_dispatch:
+
+env:
+  GH_REPO: ${{ github.repository }}
+
+permissions:
+  contents: read
+  actions: write
+
+jobs:
+  scheduled-retry:
+    if: >
+      github.event_name == 'schedule' || github.event_name == 'workflow_dispatch'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Re-run CI workflow
+        run: |
+          PR_NUMBERS=$(gh pr list -l 'ci-retry' --json number --jq '.[].number')
```

Try to use long flags where they're available for readability.
```suggestion
          PR_NUMBERS=$(gh pr list --label 'ci-retry' --json number --jq '.[].number')
```
This may need to be done elsewhere too.